helper/dockertest/dockerjobs/benchmark_run.go

```diff
@@ -0,0 +1,64 @@
+package dockerjobs
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	dockhelper "github.com/hashicorp/vault/sdk/helper/docker"
+)
+
+func CreateVaultBenchmarkContainer(t *testing.T, vaultAddr string, vaultToken string, configFile string) (func(), int64) {
+	ctx := context.Background()
+	volume := map[string]string{
+		"configs/": "/etc/",
+	}
+
+	runOpts := dockhelper.RunOptions{
+		ContainerName:   "vault-benchmark",
+		ImageRepo:       "docker.mirror.hashicorp.services/hashicorp/vault-benchmark",
+		ImageTag:        "0.1",
+		DoNotAutoRemove: true,
+		Env:             []string{fmt.Sprintf("VAULT_ADDR=%s", vaultAddr), fmt.Sprintf("VAULT_TOKEN=%s", vaultToken)},
+		CopyFromTo:      volume,
+		Cmd:             []string{"/bin/vault-benchmark", "run", fmt.Sprintf("-config=/etc/%s", configFile)},
+	}
+
+	runner, err := dockhelper.NewServiceRunner(runOpts)
+
+	if err != nil {
+		t.Fatalf("Error starting docker client for benchmark: %s", err)
+	}
+
+	service, err := runner.Start(ctx, true, false)
+	containerID := service.Container.ID
+
+	exitCh, errCh := runner.DockerAPI.ContainerWait(ctx, containerID, container.WaitConditionNotRunning)
+
+	if err != nil {
+		t.Fatalf("Error starting vault-benchmark container: %s", err)
+	}
+
+	// wait until benchmark exit
+	var exitCode int64
+	select {
+	case err := <-errCh:
+		if err != nil {
+			t.Fatal(err)
+		}
+	case status := <-exitCh:
+		statusCode := status.StatusCode
+		exitCode = statusCode
```

Comment:
```suggestion
		exitCode = status.StatusCode
```
It looks like we have an additional assignment here that we don't need.